src/main/java/de/dennisguse/opentracks/ui/intervals/IntervalsFragment.java

```diff
@@ -76,6 +76,15 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         stackModeListView = getArguments().getBoolean(FROM_TOP_TO_BOTTOM_KEY, true) ? IntervalStatisticsAdapter.StackMode.STACK_FROM_TOP : IntervalStatisticsAdapter.StackMode.STACK_FROM_BOTTOM;
         trackId = getArguments().getParcelable(TRACK_ID_KEY);
+        if (savedInstanceState != null) {
+            selectedInterval = (IntervalStatisticsModel.IntervalOption) savedInstanceState.getSerializable(SELECTED_INTERVAL_KEY);
```

Comment:
Feels like a functional change; can we put this into a separate commit? (I mean the bundle thing)

Reply:
This change is related to the migration because the new "combo box" doesn't retain the last selected value after activity recreation (old spinner did).